packages/scripts/src/config-middleware.ts

```diff
@@ -1,8 +1,13 @@
 import { SetMultiMap } from '@file-services/utils';
 import { COM, TopLevelConfig } from '@wixc3/engine-core';
 import express from 'express';
+import importFresh from 'import-fresh';
 import { IConfigDefinition } from './analyze-feature';
 
+interface ConfigModule {
```

Comment:
name it `IConfigModule` (for consistency), or just inline `as { default: TopLevelConfig }` to the `importFresh` statement.